accessibility and SEO score 100%

humailhasankhan opened the pull request.

RazzBerryMand reviewed (changes requested):
Left a link on the slack channel, make sure the metadata is in and working plz 🙏🏼

humailhasankhan commented:
> Left a link on the slack channel, make sure the metadata is in and working plz 🙏🏼

Hey Mand, I went through the link and I have changed the metadata in gatsby-config.js. I have added some default props for them as well. I have also removed the title that I was manually  passing in as props for SEO in layout. It seems to be working fine. I only needed title and description for lighthouse. I believe it is the minimum required. It seems to be working alright. Hopefully, I have fixed what was required.
